### ISSUE TITLE
missing required relation field in generated client in 0.22.1

### ISSUE THREAD

**Issue by bangbaew** [steebchen/prisma-client-go#989]
there's a problem related to #987
this is my schema
```prisma
model company {
    path     String     @id
    name     String
    image    String?
    document document[]
    access   access[]
}

model document {
    company_relation company @relation(fields: [company_path], references: [path])
    company_path     String

    parent    document?  @relation("subfolder", fields: [parent_id], references: [id], onDelete: Cascade)
    child     document[] @relation("subfolder")
    parent_id String?

    id                String    @id @default(cuid())
    created_at        DateTime  @default(now()) @db.Timestamptz(0)
    created_by        String
    updated_at        DateTime  @updatedAt @db.Timestamptz(0)
    updated_by        String?
    name              String
    type              String

    @@unique([company_path, parent_id, name])
    @@unique([id, company_path])
}
```
in 0.21.0 it generated this function
```go
func (r documentActions) CreateOne(
	_companyRelation DocumentWithPrismaCompanyRelationSetParam,
	_createdBy DocumentWithPrismaCreatedBySetParam,
	_name DocumentWithPrismaNameSetParam,
	_type DocumentWithPrismaTypeSetParam,

	optional ...DocumentSetParam,
)
```
which makes sense because company_path is a required relation field, so I could code without runtime errors.

but in 0.22.1 it generates
```go
func (r documentActions) CreateOne(
	_createdBy DocumentWithPrismaCreatedBySetParam,
	_name DocumentWithPrismaNameSetParam,
	_type DocumentWithPrismaTypeSetParam,

	optional ...DocumentSetParam,
)
```
so I have to set company_path manually, and if I don't set it, it doesn't show me any errors and let me compile,
without company_path set, it will throw this at runtime
```
pql error: Error in query graph construction: QueryParserError(ValidationError { kind: Union, message: "Unable to match input value to any allowed input type for the field. Parse errors: [`data.company_relation`: A value is required but not set, `data.company_path`: A value is required but not set]"
```

so please bring back the mandatory fields to the generated client.

**Comment by steebchen** [comment]
Can you please try the branch `fix-create`?

Do `go get github.com/steebchen/prisma-client-go@fix-create` and re-generate. Thanks!

**Comment by steebchen** [comment]
@bangbaew Feel free to join the free community discord, if you want! I'll be more responsive there as well. https://discord.gg/er3ZbmYHDk

**Comment by bangbaew** [comment]
> Can you please try the branch `fix-create`?
> 
> Do `go get github.com/steebchen/prisma-client-go@fix-create` and re-generate. Thanks!

![image](https://github.com/steebchen/prisma-client-go/assets/29559559/5392d948-5046-4245-9e5c-792969166183)
This one is fine, thank you so much!


> @bangbaew Feel free to join the free community discord, if you want! I'll be more responsive there as well. https://discord.gg/er3ZbmYHDk

Ok, I've joined the channel and will discuss in there next time.

**Comment by steebchen** [comment]
Great, thanks so much for checking!

**Comment by steebchen** [comment]
Released in [v0.22.2](https://github.com/steebchen/prisma-client-go/releases/tag/v0.22.2)
